src/devices/tuya.ts

```diff
@@ -3235,6 +3235,26 @@ const definitions: DefinitionWithExtend[] = [
          * Initially wrapped in a try catch: https://github.com/Koenkk/zigbee2mqtt/issues/6313
          */
     },
+    {
+        zigbeeModel: ['TS0041A'],
+        fingerprint: [{manufacturerName: '_TYZB01_1xktopx6'}],
```

Comment:
This can be added to the already existing `SB100` definition (it is in `cleverio.ts` but can be moved to here). Then use `tuya.whitelabel` for proper detction

Reply:
It looks like all definitions in `cleverio.ts` are in fact Tuya devices. Honestly, it exceeds my experience with these definitions to move them.